File: Backend SpringBoot/src/main/java/com/explorer/Models/MStates.java
package com.explorer.Models;

import jakarta.persistence.*;

@Entity
@Table(name = "m_states")
public class MStates {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int stateId;

    @Column(length = 100, nullable = false)
    private String state;

	public MStates(int stateId, String state) {
		super();
		this.stateId = stateId;
		this.state = state;
	}
	
	public MStates() {
		
	}

	public int getStateId() {
		return stateId;
	}

	public void setStateId(int stateId) {
		this.stateId = stateId;
	}

	public String getState() {
		return state;
	}

	public void setState(String state) {
		this.state = state;
	}

    
    
    
    
    
    
  
}

File: Bhatkanti/Models/M_States.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Bhatkanti.Models
{
    [Table("m_states")]
    public class M_States
    {
        [Key]
        public int State_ID { get; set; }

        [Required]
        [StringLength(100)]
        public string State { get; set; }
    }
}